source/administration/services/hdbpp/hdbpp-cm-interface.rst

```diff
@@ -0,0 +1,152 @@
+.. _hdbpp_cm-interface:
+
+:audience:`administrators, developers`, :lang:`C++`
+
+
+Configuration Manager interface
+-------------------------------
+
+.. contents::
+   :depth: 2
+
+.. Configuration Manager interface
+
+More in detail the device server exposes the following interface.
+
+Commands
+~~~~~~~~
+
+The commands availabile in the are summarized in commands-table.
+
++------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ArchiverAdd            | add a new instance to the archivers list; the instance must have been already created and configured via jive/astor and the device shall be running; as per release adding an device to an existing instance is not supported   |
```

Comment:
We should explicit the release in the second part of the phrase. Moreover may be rephrased as "in the current design, only one ConfigurationManager device per instance is supported"

Reply:
Actually reading this sentence, I don't understand what it means so it needs to be explicited, agreed, but I don't know how. This command is to add an eventsubscriber to the list, isn't it ? I don't understand this per release part at all… a device of what to an instance of what ?

Reply:
Indeed, these are two different points. Let's just remove the last part of the sentence. E.g. the ne one can read "add a new instance to the archivers list; the instance must have been already created and configured via jive/astor and the device shall be running"

Reply:
The other point is that, currently, you cannot deploy two devices in the same ConfigurationManager instance. For sure it is not relevant here.

Reply:
Thanks for the explanation, done.